ui/analyse/src/ceval/sunsetterProtocol.js

```diff
@@ -7,77 +7,44 @@ module.exports = function(worker, opts) {
   stopped.resolve(true);
 
   worker.send('xboard');
-
-  // Sunsetter always plays only moves right away. Count the number of played
-  // moves to show the correct mate in #n.
-  // When aiMoves is > 0 Sunsetter is in analysis mode, which uses an absolute
-  // eval score instead of relative to current color.
-  var aiMoves = 0;
-  var best, pv;
+  worker.send('reset crazyhouse');
 
   var processOutput = function(text) {
+    // console.warn("<-- %s", text);
     if (text === 'tellics stopped') {
-      aiMoves = 0;
-      best = undefined;
       if (stopped) stopped.resolve(true);
       return;
     }
     if (!work) return;
 
-    if (text.indexOf('move ') == 0) {
-      aiMoves++;
-      best = text.split(' ')[1];
-      if (work) worker.send('analyze');
-      return;
-    }
-
-    var depth, cp, mate;
+    var depth, cp, mate, best, pv;
 
-    var matches = text.match(/(\d+)\s+([-+]?\d+)\s+(\d+)\s+(\d+)\s+(([a-h1-8=@PNBRQK]+ )*[a-h1-8=@PNBRQK]+).*/);
+    var matches = text.match(/^(\d+) ([-+]?\d+) \d+ \d+ (.*)$/);
```

Comment:
I'm not sure; I haven't seen any.

How does the old regex handle that? Is the issue that 'best' could get corrupted by an exclamation mark?